### PR TITLE
issue: made pvc not required

### LINKED ISSUE #41: delete object by name and namespace only
If one wants to delete an object, for example, a VM, only name and namespace should be required, nth else. 
ATM, 
```
    - name: Delete a VM
      kubevirt_vm:
        name: test-vm
        namespace: default
        state: absent

```
returns
```
fatal: [localhost]: FAILED! => {"changed": false, "msg": "missing required arguments: pvc"}
```

and the VM is removed only when I specify `pvc`.

### PR DESCRIPTION
**Pull request by tripledes**
* Now when state is absent, pvc is not required, same behaviour as
before hiding registrydisk.
* Also moved all logic, only needed when state is present, after the
right conditional.

Fixes #41

### CODE REVIEW

File: library/kubevirt_ovm.py [hunk 1]
```diff
@@ -60,7 +60,7 @@
         description:
             - "Name of a PersistentVolumeClaim existing in the same namespace
               to use as a base disk for the Offline VM."
-        required: true
+        required: false
```
Comment (alexxa) [inline library/kubevirt_ovm.py]: @tripledes I would keep it as `true*`
`* C(pvc) option is required to create the VM. It's not needed to remove the VM.`

And the same for ovm and vmrs.

Comment (tripledes) [reply library/kubevirt_ovm.py]: @alexxa it'd be inconsistent with `argument_spec`. If DOC says it's required, it should be required by both states, `present` and `absent`, IMHO.

Comment (alexxa) [reply library/kubevirt_ovm.py]: IMHO =) If a user see 'not required' for `present` and doesn't specif it, they will see a msg: "Either pvc or registrydisk is REQUIRED". Ok.. another option, what about `required" false`, and a note in bold: 

`**C(pvc) option is required to create the VM. It's not needed to remove the VM.**`

**Comment by tripledes** [comment]
@alexxa PTAL. I've added the note and also changed the error message so `registrydisk` is not mentioned.

**Pull request merged**